.github: switch to Ubuntu 22 runner

androm3da opened the pull request.

androm3da committed:
.github: switch to Ubuntu 22 runner

Signed-off-by: Brian Cain <brian.cain@oss.qualcomm.com>

androm3da commented:
I had a feeling we'd see failures like this one:

```
qemu-system-hexagon: error while loading shared libraries: libbrlapi.so.0.7: cannot open shared object file: No such file or directory
```

... do you mind if I just disable it for now?

quic-mathbern commented:
> I had a feeling we'd see failures like this one:
> 
> ```
> qemu-system-hexagon: error while loading shared libraries: libbrlapi.so.0.7: cannot open shared object file: No such file or directory
> ```
> 
> ... do you mind if I just disable it for now?

Ok, sounds good to me. (Later, we can change the toolchain to use 22 as well and restore this here)

androm3da committed:
.github: skip the run of the tests

For now, we'll skip running QEMU because the downloaded one fails on
ubuntu 22 with:

    qemu-system-hexagon: error while loading shared libraries: libbrlapi.so.0.7: cannot open shared object file: No such file or directory

Signed-off-by: Brian Cain <brian.cain@oss.qualcomm.com>
